test/opt/dead_branch_elim_test.cpp

```diff
@@ -1334,7 +1334,7 @@ OpBranch %33
 OpStore %oc %19
 OpBranch %28
 %29 = OpLabel
-OpBranchConditional %false %27 %28
+OpBranch %27
```

Comment:
There is no longer a backedge for this loop.  That's invalid.

Reply:
This branch is the backedge (%27 contains OpLoopMerge). The exit to the merge is no longer present.

Reply:
whoops.  You're right.